v2/cmd/nuclei/main.go

```diff
@@ -113,40 +113,40 @@ on extensive configurability, massive extensibility and ease of use.`)
 	*/
 
 	flagSet.CreateGroup("input", "Target",
-		flagSet.StringSliceVarP(&options.Targets, "target", "u", []string{}, "target URLs/hosts to scan"),
+		flagSet.StringSliceVarP(&options.Targets, "target", "u", []string{}, "target URLs/hosts to scan", goflags.StringSliceOptions),
```

Comment:
What if we use `nil` instead of `[]string{}`?

Reply:
this change is unrelated to this PR and should be done separately